src/styles/block/calendar.scss

```diff
@@ -0,0 +1,33 @@
+.calendar {
+  display: flex;
+  max-width: $calendar-size;
+  flex-wrap: wrap;
+  gap: $gap-size;
+  margin: 0 auto;
+  padding: 100px 0;
+
+  &__days {
+    transition: color 0.5s, transform 0.5s;
+    position: relative;
+
+    background-color: #eee;
+    border: 1px solid black;
+
+    @include cell($squere-size);
+
+  }
+
+  &__days:hover {
+    transform: translateY(-20px);
+    background-color: pink;
+    cursor: pointer;
+  }
+  
+  &__days::before {
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
```

Comment:
Let's organize your styles
I recommend you take a look at these styles code and use it in the feature
[9elements](https://9elements.com/css-rule-order) or [htmlAcademy](https://codeguide.academy/html-css.html#css-order)